02_Spatial_data_and_R_packages_for_mapping.Rmd

```diff
@@ -0,0 +1,356 @@
+# Spatial data and R packages for mapping
+
+**Learning objectives:**
+
+- Spatial data types
+- CRS
+- making a map with R
+
+## Spatial data and R packages for mapping
+
+
+Type of Spatial data:
+
+- areal
+- geostatistical
+- point patterns
+
+```{r echo=FALSE, message=FALSE, warning=FALSE, paged.print=FALSE}
+library(tidyverse)
```

Comment:
Instead of separately listing the packages you use, a good practice is to `library()` those packages right at the top here (and get rid of the echo=FALSE). That way Rstudio will automatically check that the user has those packages installed, and it's easy to look for even without Rstudio.